PeriodicMeetings/2023-07-17.md

```diff
@@ -0,0 +1,60 @@
+# Periodic Meeting Notes
+
+## What I accomplished last week
+
+[Link to last meeting notes](./2023-07-10.md)<!-- Edit this link to the date of the last meeting -->
+
+#### B
+
+- [X] Flesh out the "Comparison With Other Packages/Ecosystems" section (later today - 2023-07-20)
```

Comment:
Doesn't look like this is live on the website though?

Reply:
Yeah, I'll bring it up today that's what I meant.